Makefile

```diff
@@ -52,6 +58,11 @@ testenv:
 	@env CONFIG_DAG=./testenv/config.json go run cmd/temporal/main.go migrate-insecure
 	make api-user
 	make api-admin
+	# create minio bucket
+	wget https://dl.minio.io/client/mc/release/linux-amd64/mc
+	chmod +x mc
+	./mc config host add $(ALIAS) $(HOST) $(ACCESSKEY) $(SECRETKEY)
+	./mc mb $(ALIAS)/$(BUCKET)
```

Comment:
can this not be done when Temporal starts up, and use the minio client library we're already using?

Reply:
I'm not sure where we would do it tbh, this bucket is expected to exist for a lot of tests and from what I can tell from reading docs this cli tool is used for a lot of basic interaction with minio from am administrator perspective.

Reply:
It can just be the same as a runmigrations option - if true, init the buckets on startup. Additional binaries are finicky - at the very least the installation here should be removed, and the user should be allowed to decide how to install mc, instead of having the hard coded Linux install here

Reply:
Ah yes good catch, i'll fix this.